packages/blockchain-wallet-v4-frontend/src/components/Flyout/index.tsx

```diff
@@ -73,7 +73,6 @@ const FlyoutModal = styled(AnimatedModal)`
   color: ${(props) => props.theme.grey700};
   ${media.mobile`
     width: 100%;
-    height: calc(100vh - 80px);
```

Comment:
why remove this? this is going to affect the height of every flyout in the app.

Reply:
mmm, this was causing the bottom gap when the modal for the backup was opened. Perhaps I can narrow the scope to just that part but I can't imagine why should we leave `80px` of gap to the bottom of the modals.

Reply:
So this is how it's now in prod. Do we want to see that gap?
![ZVSiTWI4Xb](https://user-images.githubusercontent.com/97299628/153601144-58f924da-1a80-4de3-846f-ce29fc24b11f.gif)

Here are some other flyout modals I came across, they appear to have that ugly gap.
![N2UeVAW2ZH](https://user-images.githubusercontent.com/97299628/153601544-1bc2c398-75a0-4646-98f1-f898b1987c6d.gif)